### PR TITLE
[MediaController] Add new capability APIs

### PR DESCRIPTION
**Pull request by hsgwon**
### Description of Change ###

Add new capability APIs :
- playback position
- playlist
- client custom
- search

### API Changes ###

 - TCSACR-543

### CODE REVIEW

File: src/Tizen.Multimedia.Remoting/MediaController/MediaControlServer.cs [hunk 1]
```diff
@@ -637,6 +637,82 @@ public static void SetMode360Capability(MediaControlCapabilitySupport support)
...
+        public static void SetPlaylistCapability(MediaControlCapabilitySupport support)
+        {
+            ValidationUtil.ValidateEnum(typeof(MediaControlCapabilitySupport), support, nameof(support));
+
+            Native.SetSimpleCapability(Handle, MediaControlNativeCapabilityCategory.Playlist, support).
+                ThrowIfError("Failed to set playlist capability.");
+        }
+
+        /// <summary>
+        /// Sets the <see cref="MediaControlCapabilitySupport"/> indicating client custom is supported or not.
```
Comment (wiertel) [inline src/Tizen.Multimedia.Remoting/MediaController/MediaControlServer.cs:679]: What is the "client custom" capability? Will the API user understand that?

File: src/Tizen.Multimedia.Remoting/MediaController/PlaylistCapabilityUpdatedEventArgs.cs [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        /// <since_tizen> 11 </since_tizen>
+        internal PlaylistCapabilityUpdatedEventArgs(MediaControlCapabilitySupport support)
+        {
+            ValidationUtil.ValidateEnum(typeof(MediaControlCapabilitySupport), support, nameof(support));
+
+            Support = support;
+        }
+
+        /// <summary>
+        /// Gets the value whether the repeat mode is supported or not.
```
Comment (wiertel) [inline src/Tizen.Multimedia.Remoting/MediaController/PlaylistCapabilityUpdatedEventArgs.cs:41]: wrong

**Review by wiertel — CHANGES_REQUESTED** [review]

File: src/Tizen.Multimedia.Remoting/MediaController/MediaControlServer.cs (hunk 1 above)
Comment (hsgwon) [reply src/Tizen.Multimedia.Remoting/MediaController/MediaControlServer.cs:679]: I changed it to `CustomCommand`.

File: src/Tizen.Multimedia.Remoting/MediaController/PlaylistCapabilityUpdatedEventArgs.cs (hunk 2 above)
Comment (hsgwon) [reply src/Tizen.Multimedia.Remoting/MediaController/PlaylistCapabilityUpdatedEventArgs.cs]: Fixed

File: src/Tizen.Multimedia.Remoting/MediaController/CustomCommandCapabilityUpdatedEventArgs.cs [hunk 3]
```diff
@@ -19,26 +19,27 @@
...
+        internal CustomCommandCapabilityUpdatedEventArgs(MediaControlCapabilitySupport support)
         {
             ValidationUtil.ValidateEnum(typeof(MediaControlCapabilitySupport), support, nameof(support));
 
             Support = support;
         }
 
         /// <summary>
-        /// Gets the value whether the client custom is supported or not.
+        /// Gets the value whether the custom comamnd is supported or not.
```
[the remark this answers is not in the document]
Comment (hsgwon) [reply src/Tizen.Multimedia.Remoting/MediaController/CustomCommandCapabilityUpdatedEventArgs.cs]: Fixed

**Pull request closed without merging**